### ISSUE TITLE
Automagic conversion is still present on positional arguments

### ISSUE THREAD

**Issue by nazar-pc** [yargs/yargs#1099]
The same issue as #963 (which seems to be resolved), but for positional arguments:
```javascript
console.log(yargs.command('$0 <seed>', '', function(yargs){
  yargs.positional('seed', {
    type: 'string'
  });
}).argv);
```
```
$ node test.js 0000000000000000000000000000000000000000000000000000000000000000
{ _: [], help: false, version: false, '$0': 'test.js', seed: '0' }
```

How can I get all 64 zeroes without conversion to number and back to string?

UPD: Found `"yargs": {"parse-numbers": false}` and it works, but still I believe this is an issue.

**Comment by hudovisk** [comment]
Same problem here, any tips on how to resolve it?

**Comment by nazar-pc** [comment]
@hudovisk, there is a tip in the first comment, just try it

**Comment by bcoe** [comment]
@chee @nazar-pc leading `000`s and `+`s are now treated as a special case, and should no longer result in a positional argument being parsed as a number. Please give this a try:

`npm i yargs@next`

and let me know if it does the trick.

**Comment by chee** [comment]
@nazar-pc that works perfectly, thank-you!!!!

**Comment by bcoe** [comment]
this is now available in `yargs@13.1.0`
